bundles/AdminBundle/Controller/Admin/EmailController.php

```diff
@@ -346,14 +346,37 @@ public function sendTestEmailAction(Request $request)
         }
 
         $mail = new Mail();
+
+        if($from = $request->get('from')) {
+            $mail->setFrom($from);
+        }
         $mail->addTo($request->get('to'));
         $mail->setSubject($request->get('subject'));
         $mail->setIgnoreDebugMode(true);
 
-        if ($request->get('type') == 'text') {
+        if ($request->get('emailType') == 'text') {
             $mail->setBodyText($request->get('content'));
-        } else {
+        } elseif($request->get('emailType') == 'text') {
```

Comment:
This should be `} elseif($request->get('emailType') == 'html') {` right?

Reply:
good catch :)

done